### ISSUE TITLE
pytorch code for LIDC dataset

### ISSUE THREAD

**Issue by ZhouCX117** [biomedia-mira/stochastic_segmentation_networks#6]
Hi, thanks for your excellent work! Is there a PyTorch code for the LIDC dataset?  I could not fully understand the paper, so I want to run the code to help to understand.  The PyTorch code for the 3D dataset is a little hard for me and I couldn't download the dataset. Looking award to your answer!

**Comment by MiguelMonteiro** [comment]
Hi, the code for the LIDC dataset is in tensorflow as it is a fork form the PHISEG code:
LIDC 2D. See the bottom of the readme:

The LIDC 2D results presented in the paper were generated using a [fork of the PHi-Seg code](https://github.com/MiguelMonteiro/PHiSeg-code). Evaluation script is phiseg_full_evaluation.py. Loss function in phiseg/phiseg_model.py. Model in phiseg/model_zoo/likelihoods.py. Additional configs in phiseg/experiments. You need to modify config/system.py, phiseg_full_evaluation.py and phiseg/experiments/* with your own local paths.
